hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -7934,8 +7934,9 @@ private WriteEntry doWALAppend(WALEdit walEdit, Durability durability, List<UUID
   /**
    * @return writeEntry associated with this append
    */
-  private WriteEntry doWALAppend(WALEdit walEdit, Durability durability, List<UUID> clusterIds,
-      long now, long nonceGroup, long nonce, long origLogSeqNum) throws IOException {
+  private WriteEntry doWALAppend( WALEdit walEdit, Durability durability, List<UUID> clusterIds,
```

Comment:
No change but whitespace as far as I can tell. Unintentional?

Reply:
Undid the accidental whitespace changes.